src/components/Login/SocialButtons.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import { FACEBOOK, GOOGLE, LOGIN_ERROR_DETAILS } from 'common/constants';
+import { orcidAuthAppId } from 'common/injectGlobals';
+import FacebookLogin from 'components/loginButtons/FacebookLogin';
+import GoogleLogin from 'components/loginButtons/GoogleLogin';
+import OrcidLogin from 'components/loginButtons/OrcidLogin';
+import ExternalLink from 'uikit/ExternalLink';
+import { PromptMessageContainer, PromptMessageContent } from 'uikit/PromptMessage';
+
+import { facebookLogin, googleLogin } from '../../services/login';
+import { Provider } from '../../store/userTypes';
+import { Nullable } from '../../store/utilityTypes';
+import { Box } from '../../uikit/Core';
+
+// @ts-ignore
+import { loginError } from './Login.module.css';
+
+type Props = {
+  disabled: boolean;
+  handleToken: ({
+    provider,
+    handler,
+    token,
+  }: {
+    provider: Provider;
+    handler: (t: string) => Promise<any>;
+    token: string;
+  }) => Promise<void>;
+  handleError: (e: string) => void;
+  thirdPartyDataError: Nullable<Error>;
+  unknownError: Nullable<Error>;
+};
+
+const SocialButtons = (props: Props) => {
+  const { disabled, handleToken, handleError, thirdPartyDataError, unknownError } = props;
+  const orcidLoginEnabled = Boolean(orcidAuthAppId);
+
+  const getErrorMessage = () => {
+    if (unknownError) {
+      return (
+        <>
+          Uh oh, looks like something went wrong.
+          {/* @ts-ignore */}
+          <ExternalLink
+            hasExternalIcon={false}
+            href="https://kidsfirstdrc.org/contact"
+            target="_blank"
+          >
+            &nbsp;Contact us&nbsp;
+          </ExternalLink>
+          and we will help investigate why you are unable to sign in.
+        </>
+      );
+    } else if (thirdPartyDataError) {
+      return LOGIN_ERROR_DETAILS.thirdPartyData;
+    } else {
+      return LOGIN_ERROR_DETAILS.facebook;
+    }
+  };
```

Comment:
since you are not dependent of the render cycle. I would put it outside in orde to be create only once in page life time. Instead of being add to memory each time its' rendered and then garbage collected